### PR TITLE
PLAT-128519: Support any valid CSS duration for any `type` prop in ui/Transition

### PR DESCRIPTION
**Pull request by ybsung**
Enact-DCO-1.0-Signed-off-by: YB Sung (yb.sung@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)

The ui/Transition with `fade` or `slide` type prop supports only preset duration (`short`, `medium`, and `long`).  It is needed to support any valid CSS duration for them.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

Support any valid CSS duration for any `type` prop in ui/Transition

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)

PLAT-128519

### CODE REVIEW

File: packages/ui/Transition/Transition.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ const forwardTransitionEnd = forward('onTransitionEnd');
 const forwardOnShow = forward('onShow');
 const forwardOnHide = forward('onHide');
 
+const validateDuration = (duration) => (typeof duration === 'number' ? duration + 'ms' : duration);
```
Comment (jeonghee27) [inline packages/ui/Transition/Transition.js:34]: validatedDuration  ?? 

@MikyungKim @0x64  What do you think

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Comment by 0x64** [comment]
@ybsung I just realized that you added a change log to a wrong file. :) Please move your log to the proper change log file.

**Comment by ybsung** [comment]
@0x64 

> I just realized that you added a change log to a wrong file. :) Please move your log to the proper change log file.
I fixed it. Thanks for letting me know it.

**Pull request merged**